tests/adapters/test_protocol.py

```diff
@@ -170,11 +177,16 @@ def test_read_without_write(self):
         with raises(AssertionError):
             a.read_bytes(3)
 
-    def test_read_empties_read_buffer(self):
+    def test_read_all_bytes_empties_read_buffer(self):
         a = ProtocolAdapter()
         a._read_buffer = b"jklasdf"
-        a.read_bytes(10)
-        assert a._read_buffer == b""
+        a.read_bytes(7)
+        assert a._read_buffer is None
+
+    def test_read_all_bytes__from_pairs_empties_read_buffer(self):
```

Comment:
Two consecutive underscores in the test name, probably a typo.

Reply:
Thanks, fixed.